js/templates/components/moderators/moderators.html

```diff
@@ -1,12 +1,5 @@
 <div class="moderatorsWrapper clrBr <% print(ob.wrapperClasses) %> js-moderatorsWrapper">
-  <% // placeholder so border collapse doesn't erase the table border when there are no table cells %>
-  <% if (ob.placeholder) { %>
-    <div class="moderatorCard clrBr">
-      <div class="moderatorCardInner">
-      </div>
-    </div>
-  <% } %>
-  <% if (ob.totalIDs && !ob.totalShown) { %>
```

Comment:
The placeholder got refactored so the messages below take care of everything with less duplication.